lms/templates/seq_module.html

```diff
@@ -58,9 +62,21 @@
       <span class="icon fa fa-chevron-prev" aria-hidden="true"></span>
       <span>${_('Previous')}</span>
     </button>
-    <button class="sequence-nav-button button-next">
+      <button class="sequence-nav-button button-next">
       <span>${_('Next')}</span>
       <span class="icon fa fa-chevron-next" aria-hidden="true"></span>
     </button>
   </nav>
+  % if not next_url:
+  <form method="post" action="${reverse('end_course')}">
```

Comment:
In my opinion, We can add the `end_course` url to the  params dict in the `_student_view` method: https://github.com/raccoongang/edx-platform/blob/add_button_end_course/common/lib/xmodule/xmodule/seq_module.py#L313   
After that, we can use `end_course` variable in this seq_module.htm.  I think this is a better way.

Reply:
I don’t see the point, since this variable `end_course` will be used once and this will litter the code.